docs/developer/payments.mdx

```diff
@@ -669,7 +669,7 @@ The webhook will be sent only to the app that is the owner of the transaction.
 
 :::
 
-##### Asynchronously processing the action
```

Comment:
This looks like an unrelated change.

Reply:
It is, as h5 fives should not be used as we don't differentiate h5 and h4, and those are actually typos and should be h4